app/models/explainer.rb

```diff
@@ -61,7 +61,7 @@ def self.update_paragraphs_in_alegre(id, previous_paragraphs_count, timestamp)
 
     base_context = {
       type: 'explainer',
-      team: explainer.team.slug,
+      team: explainer.team_id,
```

Comment:
Let's rename `team` to `team_id` to be consistent with how we index fact-checks?
https://github.com/meedan/check-api/blob/819bc4a3d3a834de2be6f5bee5e50e69ef996b76/app/models/concerns/alegre_v2.rb#L264

Reply:
Since I was still here... done!